superduperdb/backends/base/artifact.py

```diff
@@ -133,8 +149,11 @@ def load_artifact(self, r):
         if file_id is None:
             assert uri is not None, '"uri" and "file_id" can\'t both be None'
             file_id = _construct_file_id_from_uri(uri)
-        bytes = self._load_bytes(file_id)
-        return datatype.decoder(bytes)
+        if r['datatype'] == 'file':
```

Comment:
This is a typo right? `"datatype"` is already being used.

Reply:
Should be `"type"`.

Reply:
The key `"datatype"` is the same as the key `"datatype"` in line 146 `datatype = self.serializers[r['datatype']]`